src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -189,6 +189,31 @@ storiesOf('Dropdown', module)
       </Legends>
     </Wrapper>
   ))
+  .add('rta', () => (
```

Comment:
I'm sorry! This change of the stories is a sample for the problem and does not need for master, so please revert my commit (b3a5aebc0b1182dee635bf54797f7841b0639fd3).

Reply:
@wmoai Thank you! I've reverted the commit 👍